app/src/main/java/edu/ucsc/cmps115_spring2017/face2name/MainScreen.java

```diff
@@ -59,5 +66,43 @@ public void onPreviewReady() {
         mCameraPreview.startPreview();
     }
 
+
+
+    public boolean onTouchEvent(MotionEvent event) {
+        int l_x = mLayerView.getRectPoint("Left X");
+        int r_x = mLayerView.getRectPoint("Right X");
+        int u_y = mLayerView.getRectPoint("Upper Y");
+        int l_y = mLayerView.getRectPoint("Lower Y");
+
+        Log.e("PRESS", "LX: " + l_x + " RX: " + r_x + "UY: " + u_y + "LY: " + l_y);
+
+        int touchX = (int) event.getX();
+        int touchY = (int) event.getY();
+        Log.e("PRESS", "GET_X: "+touchX+ " GET_Y: "+touchY);
+        Log.e("PRESS", "EVENT ACTION: "+event.getAction()+ " ACTION BUTTON PRESS: "+ MotionEvent.ACTION_BUTTON_PRESS);
```

Comment:
Log.e() is for error messages. Additionally, calls to this function are not compiled out when making a release build. Log.d(), on the other hand, is expressly made for debug messages, and the calls are compiled out on release builds.